lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -120,6 +120,35 @@ impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
         }
         unique_record_ids.len()
     }
+
+    /// Plain search not using the inverted index.
+    pub fn search_plain(
+        &self,
+        vectors: &[&QueryVector],
+        filter: &Filter,
```

Comment:
I agree with the filter. plain search is applicable only when filtering is enabled